src/v2/Apps/Conversation/Components/__tests__/ConversationMessages.jest.tsx

```diff
@@ -1,90 +1,89 @@
               },
-              {
-                node: {
-                  __typename: "Message",
-                  id: "456",
-                  internalID: "456",
-                  body: "hello world",
-                  isFromUser: true,
-                  createdAt: Date.now().toString(),
-                  attachments: [],
-                },
-              },
-            ],
-          } as any
-        }
-        events={
+            },
+          },
           {
-            edges: [
-              {
-                node: {
-                  buyerAction: "OFFER_ACCEPTED_CONFIRM_NEEDED",
-                  internalID: "7adde1e2-bdd4-4360-9484-989d6dd3248esss",
-                  id: "7adde1e2-bdd4-4360-9484-989d6dd3248esss",
-
-                  orderHistory: [
-                    {
-                      __typename: "CommerceOfferSubmittedEvent",
-                      internalID: "7adde1e2-bdd4-4360-9484-989d6dd3248e",
-                      createdAt: Date.now().toString(),
-                      state: "PENDING",
-                      stateReason: null,
-                      offer: {
-                        amount: "£40,000",
-                        definesTotal: true,
-                        fromParticipant: "SELLER",
-                        offerAmountChanged: false,
-                        respondsTo: {
-                          fromParticipant: "BUYER",
-                        },
-                      },
-                    },
-                    {
-                      __typename: "CommerceOrderStateChangedEvent",
-                      internalID: "7adde1e2-bdd4-4360-9484-989d6dde",
+            __typename: "CommerceOrderStateChangedEvent",
+            internalID: "7adde1e2-bdd4-4360-9484-989d6dde",
 
-                      createdAt: Date.now().toString(),
-                      state: "APPROVED",
-                      stateReason: null,
-                      offer: {
-                        amount: "£40,000",
-                        definesTotal: true,
-                        fromParticipant: "SELLER",
-                        offerAmountChanged: false,
-                        respondsTo: {
-                          fromParticipant: "BUYER",
-                        },
-                      },
-                    },
-                  ],
-                },
+            createdAt: Date.now().toString(),
+            state: "APPROVED",
+            stateReason: null,
+            offer: {
+              amount: "£40,000",
+              definesTotal: true,
+              fromParticipant: "SELLER",
+              offerAmountChanged: false,
+              respondsTo: {
+                fromParticipant: "BUYER",
               },
-            ],
-          } as any
-        }
-      />
-    )
-    expect(wrapper.find("Message").length).toBe(2)
-    expect(wrapper.find("OrderUpdate").length).toBe(2)
+            },
+          },
+        ],
+      },
+    },
+  ],
+} as any
```

Comment:
(nonblocking) - with our relay test tools we can lean on auto fixtures and only mock out those properties that we need to test behavior of the component. Aware that this stuff was here before, but the less fixture data the better.